Update create-token.mdx (#17040)
cloudflare/cloudflare-docs: src/content/partials/fundamentals/create-token.mdx

:::

> 1. Determine if you want a user token or an account owned token. If you are developing a new service that you want multiple superadministrators to use, and the endpoints you are calling are compatible with Account Owned Tokens, the option exists to use account tokens that are not connected to a specific user

Reviewer: ```suggestion
1. Determine if you want a user token or an account owned token. If you are developing a new service that you want multiple superadministrators to use and the endpoints that you are calling are compatible with account owned tokens, the option exists to use account tokens that are not connected to a specific user.
```

2. From the [Cloudflare dashboard](https://dash.cloudflare.com/profile/api-tokens/), go to **My Profile** > **API Tokens** for user tokens. For Account Tokens, go to **Manage Account** > **API Tokens**.

3. Select **Create Token**.

> 4. Select a template from the available [API token templates](/fundamentals/api/reference/template/) or create a custom token. We use the **Edit zone DNS** template in the following examples.

Reviewer: ```suggestion
4. Select a template from the available [API token templates](/fundamentals/api/reference/template/) or create a custom token. The following example uses the **Edit zone DNS** template.
```
